src/routes/event.js

```diff
@@ -0,0 +1,13 @@
+const { mockEvents } = require('../../database/events_schema.js');
+
+module.exports = (req, res) => {
+  const id = req.params.id;
```

Comment:
lines 4 to 9 is what im talking about, take this code and put it inside a function in a different file